certificate/urls.py

```diff
@@ -38,8 +38,8 @@
     url(r'^scipy_download_2016/$', 'scipy_download_2016', name='scipy_download_2016'),
     url(r'^openfoam_symposium_download_2016/$', 'openfoam_symposium_download_2016', name='openfoam_symposium_download_2016'),
     url(r'^openfoam_symposium_feedback_2016/$', 'openfoam_symposium_feedback_2016', name='openfoam_symposium_feedback_2016'),
-    url(r'^fossee_internship_cerificate_download/$', 'fossee_internship_cerificate_download', name='fossee_internship_cerificate_download'),
-    url(r'^fossee_internship16_cerificate_download/$', 'fossee_internship16_cerificate_download', name='fossee_internship16_cerificate_download'),
+    url(r'^fossee_internship15_cerificate_download/$', 'fossee_internship_cerificate_download', name='fossee_internship15_cerificate_download'),
+    url(r'^fossee_internship_cerificate_download/$', 'fossee_internship16_cerificate_download', name='fossee_internship_cerificate_download'),
```

Comment:
I see inconsistency in the names, which slightly confuses, also same views for two URLs but different name!

Reply:
I will comment the 41st line, since we are not using it..is that ok?

Reply:
Simply remove it.

Reply:
okay